app/ui/onionskin.ui

```diff
@@ -0,0 +1,343 @@
+        </property>
+        <property name="checkable">
+         <bool>true</bool>
+        </property>
+        <property name="autoRaise">
+         <bool>true</bool>
+        </property>
+       </widget>
+      </item>
+     </layout>
+    </item>
+    <item>
+     <widget class="QLabel" name="onionOpacityLabel">
+      <property name="text">
+       <string>Distributed Opacity</string>
+      </property>
+      <property name="alignment">
+       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
+      </property>
+     </widget>
+    </item>
+    <item>
+     <layout class="QHBoxLayout" name="opacityLayout">
+      <item>
+       <layout class="QVBoxLayout" name="minimumLayout">
+        <item>
+         <widget class="QLabel" name="onionMinOpacityLabel">
+          <property name="text">
+           <string>Min.%</string>
+          </property>
+          <property name="alignment">
+           <set>Qt::AlignCenter</set>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QSpinBox" name="onionMinOpacityBox">
+          <property name="minimumSize">
+           <size>
+            <width>50</width>
+            <height>0</height>
+           </size>
+          </property>
+          <property name="maximumSize">
+           <size>
+            <width>50</width>
+            <height>16777215</height>
+           </size>
+          </property>
+          <property name="alignment">
+           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
+          </property>
+          <property name="maximum">
+           <number>100</number>
+          </property>
+         </widget>
+        </item>
+       </layout>
+      </item>
+      <item>
+       <layout class="QVBoxLayout" name="maximumlLayout">
+        <item>
+         <widget class="QLabel" name="onionMaxOpacityLabel">
+          <property name="text">
+           <string>Max.%</string>
+          </property>
+          <property name="alignment">
+           <set>Qt::AlignCenter</set>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QSpinBox" name="onionMaxOpacityBox">
+          <property name="minimumSize">
+           <size>
+            <width>50</width>
+            <height>0</height>
+           </size>
+          </property>
+          <property name="maximumSize">
+           <size>
+            <width>50</width>
+            <height>16777215</height>
+           </size>
+          </property>
+          <property name="alignment">
+           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
+          </property>
+          <property name="maximum">
+           <number>100</number>
+          </property>
+         </widget>
+        </item>
+       </layout>
+      </item>
+     </layout>
+    </item>
+    <item>
+     <widget class="QCheckBox" name="onionSkinMode">
+      <property name="text">
+       <string>Skip Exposure Frames</string>
```

Comment:
I understand what you mean. My main problem with the word `empty ` is that while it is visually intuitive in the context of the interface, it misrepresents what the user is seeing in the context of the animation workflow and how the film medium works overall.

How about we change it to **Show Keyframes Only** instead? That shouldn't be difficult to understand, I think 🤔

Reply:
it's a balance yes, we try to make the interface understandable from a layman point of view, but if we make up the interface based on terms that you would only know if you came from the the industry, then it becomes more difficult to use. The exposure keyframe is a bit abstract in pencil2D, it's there implicitly because we say it's there, but there's no visual representation to back that up (yet), it's automatic, therefore I think it's not a good idea to use a wording that we're not using anywhere else and the user won't be able to figure out what is unless they know it already or read a manual. 

Empty can be understood by the layman because as you said, it's visually intutitive, which imo is more important than it's correct.

Reply:
@CandyFace Hmm well but I wrote the proposal as plural > "Show **Keyframe(s)** Only" to refer to all the keyframes. I really don't get any intent of there being different "kinds" of keyframes, but alright. 

I'll revert to **empty frames** to close this, but just so you know I'm not okay with it for more reasons that I can fit in this text field 😝 

I just want this to be merged soon so the other pending reviews can be done in due time as well. Thanks again for taking the time to review 👍